app/Http/Controllers/Store/Auth/ForgotPasswordController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers\Service\Auth;
```

Comment:
Will it make more sense to use Store and Admin and API)? rather than Service? Is Service Admin?

Reply:
I totally forgot to alter these